dask_kubernetes/operator/deployment/helm/dask-kubernetes-operator/values.yaml

```diff
@@ -85,3 +85,7 @@ metrics:
         - dask.org/cluster-name
         - dask.org/workergroup-name
       metricRelabelings: [] # MetricRelabelConfigs to apply to samples before ingestion.
+
+workerAllocationSettings:
+  size: "5"
+  delay: 1s
```

Comment:
Can we default these to `nil` so that we don't batch by default? And add a comment to the end of the line explaining how to use each option.